proposal/nctu16028/README.md

```diff
@@ -0,0 +1,57 @@
+# Exploring Bezier Curve
+
+## Basic Information
+GitHub repository: https://github.com/nctu16028/Exploring-Bezier-Curve
+
+## Problem to Solve
+A **Bezier curve** is a parametric curve that is widely used in computer graphics and related fields.
+![Recursive definition](assets/def.png)
+
+An algorithm called **De Casteljau's algorithm** calculates Bezier curves using this definition.
+
+In this project, I am going to build a system that receives some two-dimensional points and then presents the formation of Bezier curve in animation. Besides, the users can decide whether to show additional information of the curves. For example, tangent, curvature, or bounding box, etc.
+
+## Prospective Users
+1. Beginners of numerical analysis or computer graphics
+2. Anyone simply appreciating the beauty of Bezier curves
+
+## System Architecture
+![Flow chart](assets/flow.png)
+
+## API Description
+**Python**
+- `main()`: parse input parameters, including order, coordinates, and flags of showing additional information
+- `draw_animation()`: call subroutines and then consolidate the results to produce a gif animation
+
+**C++**
+- `draw_curve()`: calculate the coordinates of curve points at a given time frame
```

Comment:
It will be good to think about how to handle many Bezier curves at once (batch operations).  It has great impact to performance, which should be measured.